components/multiselect/multiselect.jsx

```diff
@@ -191,10 +191,24 @@ export default class MultiSelect extends React.Component {
         this.props.handleDelete(values);
     }
 
+    defaultAriaLabelRenderer(option) {
+        if (!option) {
+            return null;
+        }
+        return option.label;
+    }
+
     render() {
         const options = Object.assign([], this.props.options);
         const {totalCount, users, values} = this.props;
 
+        let renderer;
+        if (this.props.ariaLabelRenderer) {
```

Comment:
It'd be easier to define this as a `defaultProp` on the component, so that it's automatically passed as `this.props.ariaLabelRenderer` when undefined (but not null) is otherwise passed. Here's an example of another component that does it: https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/badges/badge.jsx#L16